xml/System.Numerics/Complex.xml

```diff
@@ -1806,10 +1942,24 @@
         <Parameter Name="value" Type="System.UInt32" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The value to convert to a complex number.</param>
+        <summary>Defines an implicit conversion of a 32-bit unsigned integer to a complex number.   
+           
+This API is not CLS-compliant.</summary>
+        <returns>An object that contains the value of the <paramref name="value" /> parameter as its real part and zero as its imaginary part.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The overloads of the <xref:System.Numerics.Complex.op_Implicit%2A> operator define the types from which a compiler can automatically convert a <xref:System.Numerics.Complex> object without an explicit casting operator (in C#) or a call to a conversion function (in Visual Basic). They are widening conversions that do not involve data loss and do not throw an <xref:System.OverflowException>.
+
+ This overload lets the compiler handle conversions from an unsigned 32-bit integer to a complex number, as the following example shows. Note that the result of the conversion is a complex number whose real part is equal to the unsigned 32-bit integer and whose imaginary part is equal to zero.
+
+ [!code-csharp[System.Numerics.Complex.Implicit#9](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.implicit/cs/implicit1.cs#9)]   
```

Comment:
The C# code is incorrect. Could you change it, Tom? In its present form:
```
private static void ComplexFromUInt32()
   {
      // <Snippet9>
      int intValue = 197461;
      System.Numerics.Complex c1 = intValue;
      Console.WriteLine(c1);
      // The example displays the following output:
      //       (197461, 0)
      // </Snippet9>
   }
```
Change
``` 
int intValue = 197461;
```
to
```
int intValue = 197461;
```
The VB code is OK.